lonboard/_layer.py

```diff
@@ -183,6 +183,20 @@ def _add_extension_traits(self, extensions: Sequence[BaseExtension]):
     - Default: `False`
     """
 
+    selected_bounds = traitlets.List(
+        trait=traitlets.Float(allow_none=False),
+        allow_none=True,
+        default_value=None,
+        sync=True,
```

Comment:
This needs to be `.tag(sync=True)`; I don't think you can pass it in here as you have